tests/build/build.go

```diff
@@ -123,6 +126,190 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 		})
 
+		When("a new simple build component is created", Label("annotations"), func() {
+			var component *appservice.Component
+
+			BeforeAll(func() {
+				cdq, err := f.AsKubeDeveloper.HasController.CreateComponentDetectionQuery(annotationsComponentName, testNamespace, helloWorldComponentGitSourceURL, "", "", "", false)
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(len(cdq.Status.ComponentDetected)).To(Equal(1))
+
+				for _, compDetected := range cdq.Status.ComponentDetected {
+					compDetected.ComponentStub.ComponentName = annotationsComponentName
+					component, err = f.AsKubeAdmin.HasController.CreateComponentFromStub(compDetected, testNamespace, "", "", applicationName)
+					Expect(err).NotTo(HaveOccurred())
+				}
+			})
+
+			It("initial build triggers a pipelinerun", func() {
+				timeout = time.Minute * 1
+				interval = time.Second * 1
+
+				Eventually(func() bool {
+					pipelineRun, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, "")
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun has not been created yet")
+						return false
+					}
+					return pipelineRun.HasStarted()
+				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start")
+
+				Expect(f.AsKubeAdmin.TektonController.DeleteAllPipelineRunsInASpecificNamespace(testNamespace)).To(Succeed())
+			})
+
+			It("simple build can be triggered manually", func() {
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				component.Annotations[build.BuildRequestAnnotationName] = build.BuildRequestTriggerSimpleBuildAnnotationValue
+				Expect(f.AsKubeAdmin.CommonController.KubeRest().Update(context.TODO(), component)).To(Succeed())
+
+				Eventually(func() bool {
+					pipelineRun, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, "")
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun has not been created yet")
+						return false
+					}
+					return pipelineRun.HasStarted()
+				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start")
```

Comment:
Shall we check the status of the pipelinerun to be completed successfully for `trigger-simple-build` ?

Reply:
I see in the old annotation test we are checking for successful completion. Since these tests are going to replace the old annotation tests in the future and tough it will takes more time test to the completion of the PR and test  execution time increases, I think it might worth checking for successful completion.